package.json

```diff
@@ -132,7 +132,7 @@
   },
```

Comment:
The `scripts` area in the `package.json` still mentions Node 14:

https://github.com/renovatebot/renovate/blob/0ab15a4b214fb16f69f8c8ece9ae81b438a650f7/package.json#L26

Reply:
will cleanup later, as it doesn't change anything